apps/solr.go

```diff
@@ -52,3 +53,76 @@ func (r MetricsReceiverSolr) Pipelines() []otel.Pipeline {
 func init() {
 	confgenerator.MetricsReceiverTypes.RegisterType(func() confgenerator.Component { return &MetricsReceiverSolr{} })
 }
+
+type LoggingProcessorSolrSystem struct {
```

Comment:
On the metrics side, typically there is [only one receiver needed](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/confgenerator/testdata/invalid/linux/metrics-receiver_unsupported_type/golden_error). On the logging side, sometimes [multiple receivers are needed](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/confgenerator/testdata/invalid/linux/logging-receiver_unsupported_type/golden_error) (one per a distinct logging format). e.g.:

`nginx_access` and `nginx_error`
`elasticsearch_gc` and `elasticsearch_json`

One benefit of `solr_system` is that in case there are other format we need to support in the future, we can name it `solr_XXX`.

For the previous apps we released, `postgresql_general` and `cassandra_system` seem to follow a similar pattern.

Reply:
Makes sense, thanks!